src/sentry/static/sentry/app/components/issueLink.jsx

```diff
@@ -11,10 +12,11 @@ import {t} from 'app/locale';
 
 export default class IssueLink extends React.Component {
   static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
```

Comment:
Do we need both `organization` and `orgId`?

Reply:
I think it's better to leave orgId so we can more easily reverse out `organization` when we remove the feature toggle